.vale/fixtures/AsciiDoc/LinkContainsLinkText/testinvalid.adoc

```diff
@@ -0,0 +1,2 @@
+//vale-fixture
```

Comment:
Create a //vale-fixture for every entry in your rule please. That way we know the rule works for everything you want to catch